static/js/settings_account.js

```diff
@@ -53,10 +53,10 @@ function settings_change_success(message) {
     ui_report.success(message, $('#account-settings-status').expectOne());
 }
 
-function update_user_custom_profile_fields(fields) {
+function update_user_custom_profile_fields(fields, method=channel.patch) {
```

Comment:
This doesn't work in JS, I think

Reply:
From ES6/ES2015, default parameters is in the language specification. But I did change it anyway.